ceph-releases/kraken/ubuntu/16.04/daemon/Dockerfile

```diff
@@ -29,7 +29,7 @@ ADD https://bin.equinox.io/c/ekMN3bCZFUn/forego-stable-linux-amd64.tgz /forego.t
 
 
 # install prerequisites
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y wget unzip uuid-runtime python-setuptools udev runit sharutils python3 dmsetup && \
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y wget unzip uuid-runtime python-setuptools udev sharutils python3 dmsetup && \
```

Comment:
I think we might be able to remove sharutils and python3, let's test and see how the CI reacts.